docker/generic/start_proxy.py

```diff
@@ -442,7 +422,6 @@ def make_argparser():
         help="The Google project id for Stack driver tracing")
     parser.add_argument(
         '--tracing_sample_rate',
-        default=0.001,
         help='''
         Tracing sampling rate from 0.0 to 1.0.
         By default, 1 out of 1000 requests are sampled.
```

Comment:
Instead of 0.001, so what is the default sample rate now?

Reply:
It's still 0.001. It's set in config generator flags instead, and python will just not pass any value if the user doesn't override it.

This is a pattern @JLXIA and I have discussed in the past. Keep the python flags empty, and place the defaults in the config generator flags. Otherwise the defaults will be duplicated across binaries and can lead to problems in the future.

Added a test case for the default flag value in config generator.